src/hotspot/cpu/x86/gc/g1/g1BarrierSetAssembler_x86.cpp

```diff
@@ -24,7 +24,7 @@
 
 #include "asm/macroAssembler.inline.hpp"
 #if INCLUDE_CDS
-#include "code/SCCache.hpp"
```

Comment:
While you are at it, remove `INCLUDE_CDS` guards around includes like these?

Reply:
Will do.